modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/InputVariable.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.credential;
+
+/**
+ * The class object for the CredentialInput.
+ */
+public class InputVariable {
+
+    /**
+     * The name of the InputVariable.
+     */
+    String name;
+
+    /**
+     * The description of the InputVariable.
+     */
+    String description;

```

Comment:
both  here and in Credential class, we need another name instead of description. 
Probably even an Enum is better here.

Reply:
I feel `description` is good here. It is only some helpful messages for the document and the end-user, letting them know what is it defined for and how to use it.
`Enum` not make sense, or you misunderstood my ideas.